moz_security/CMakeLists.txt

```diff
@@ -22,3 +36,8 @@ foreach(test IN LISTS integration_tests)
     CONFIGURATIONS integration
     WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/integration/${test})
 endforeach()
+
+set(EMPTY_DIR ${CMAKE_BINARY_DIR}/empty)
+file(MAKE_DIRECTORY ${EMPTY_DIR})
+install(DIRECTORY ${EMPTY_DIR}/ DESTINATION ${INSTALL_IOMODULE_PATH} ${DPERMISSION})
+install(TARGETS irc DESTINATION ${INSTALL_IOMODULE_PATH})
```

Comment:
Seems OK here? I don't see any reason to add a different one

Reply:
Hmm, if I wanted to use an IRC alert I don't think the moz_security module would be the first place I would look.  I think it would make sense to just have a top level IRC module/dir as it is not related to moz_security.